initCobraToolbox.m

```diff
@@ -317,9 +322,9 @@ function initCobraToolbox()
     SOLVERS.gurobi.type = {'LP', 'MILP', 'QP', 'MIQP'};
     SOLVERS.ibm_cplex.type = {'LP', 'MILP', 'QP'};
     SOLVERS.matlab.type = {'LP', 'NLP'};
-    SOLVERS.mosek.type = {'LP', 'QP', 'MILP'};
+    SOLVERS.mosek.type = {'LP', 'QP'};
     SOLVERS.pdco.type = {'LP', 'QP'};
-    SOLVERS.quadMinos.type = {'LP', 'NLP'};
+    SOLVERS.quadMinos.type = {'LP'};
```

Comment:
Why?

Reply:
because quadMinos is not an NLP solver?

Reply:
It can, but it is not yet implemented (I think).